libs/libc/modlib/modlib_globals.S

```diff
@@ -16,7 +16,11 @@
 	.type	SYMBOL(\ep), "object"
 	.endm
 	.macro SIZE ep
+#  if defined(__ghs__)
+	.size	SYMBOL(ep), . - SYMBOL(ep)
+#  else
 	.size	SYMBOL(\ep), . - SYMBOL(\ep)
```

Comment:
ditto

Reply:
> ditto

I have tried the solution from comments, but build error still report out